pygments/lexers/asc.py

```diff
@@ -0,0 +1,49 @@
+"""
+    pygments.lexers.asc
+    ~~~~~~~~~~~~~~~~~~~
+
+    Lexer for various ASCII armored files.
+
+    :copyright: Copyright 2021 by the Pygments team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+import re
+
+from pygments.lexer import RegexLexer, bygroups
+from pygments.token import Comment, Generic, Name, Operator, Text, String
+
+__all__ = ['AscLexer']
+
+
+class AscLexer(RegexLexer):
+    """
+    Lexer for ASCII armored files, containing `-----BEGIN/END ...-----` wrapped base64 data.
+    """
+    name = 'ASCII armored'
+    aliases = ['asc', 'pem']
```

Comment:
Should we add some more here?

Reply:
As long as they're not ambiguous it's fine.

Reply:
I was thinking about certificates and the like, but as/if we're not adding filenames and mimetypes for them, I think it's ok to leave them out from here, too.